Dockerfiles/agent/README.md

```diff
@@ -281,7 +281,7 @@ docker run -d --name datadog-agent \
            -e DD_API_KEY=<YOUR_API_KEY> \
            -e DD_LOGS_ENABLED=true \
            -e DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL=true \
-           -e DD_AC_EXCLUDE="name:dd-agent" \
+           -e DD_AC_EXCLUDE="name:datadog-agent" \
```

Comment:
Could we exclude the agent based on its image name (`image:datadog/agent.*`) to avoid being dependent on the container name?
Filtering on the container name would not work with orchestrators

Reply:
Also, so we have a plan to remove it when 6.6 is out?
This option means clients don't have metrics on the agent containers anymore, as the docker and kube checks honor that filter

Reply:
We only publicise it alongside `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` and we had a note below mentionning that it removes the monitoring for the agent logs.

If this becomes an issue. We would then need to figure out internally if we should have a specific behaviour to exclude Log and/or Metric only.

We can definitely exclude it by image or name, the advantages of name is that it does not use wildcard and therefore reduce the CPU impact but maybe filtering by name is more cpu consuming than by image.

Thoughts?

Reply:
@xvello to make sure you saw the answer :)